Doc/whatsnew/3.12.rst

```diff
@@ -998,9 +998,9 @@ Optimizations
 CPython bytecode changes
 ========================
 
-* Remove the :opcode:`LOAD_METHOD` instruction. It has been merged into
+* Remove the :opcode:`!LOAD_METHOD` instruction. It has been merged into
```

Comment:
Sorry, my rST is rusty. What does `!` do?

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-syntax